composer.json

```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
+        "php": ">=5.5",
```

Comment:
We will likely need to upgrade dependency of SlmQueue too. That's why I'd love if we could start our work on SlmQueue 0.5 so we can tag sometimes and upgrade all the other modules.